newrelic-agent/src/main/java/com/newrelic/agent/config/ApplicationLoggingForwardingConfig.java

```diff
@@ -22,11 +25,21 @@
 
     public ApplicationLoggingForwardingConfig(Map<String, Object> props, String parentRoot, boolean highSecurity) {
         super(props, parentRoot + ROOT + ".");
-        maxSamplesStored = getProperty(MAX_SAMPLES_STORED, DEFAULT_MAX_SAMPLES_STORED);
+        maxSamplesStored = tryToGetMaxSamplesStoredOrDefaultIfValueNotInteger();
```

Comment:
Can we change the name of this method to `initMaxSamplesStored` as that is a naming pattern that we use a lot throughout the codebase. Otherwise looks good!

Reply:
Done